RestaurantsWeb/Models/AddRestaurantRequestValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using RestaurantsWeb.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RestaurantsWeb.Models
+{
+    public class AddRestaurantRequestValidator : AbstractValidator<AddRestaurantRequest>
+    {
+        public AddRestaurantRequestValidator()
+        {
+            CascadeMode = CascadeMode.StopOnFirstFailure;
```

Comment:
Please remove this line, or leave it only if you think it should fail at first error that it finds